Sources/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+extension UIViewController {
+  func add(childViewController: UIViewController) {
+    childViewController.willMove(toParentViewController: self)
+    addChildViewController(childViewController)
+    view.addSubview(childViewController.view)
+    layout(childViewController: childViewController)
+    childViewController.didMove(toParentViewController: self)
+  }
+
+  func remove(childViewController: UIViewController) {
+    childViewController.willMove(toParentViewController: nil)
+    childViewController.view.removeFromSuperview()
+    childViewController.removeFromParentViewController()
+  }
+
+  private func layout(childViewController: UIViewController) {
```

Comment:
- Not all child view controller wants full layout ?
- I think child view controller has the same size as its parent by default ?
- At `viewDidLoad`, the parent `view.bounds` is not correct

Reply:
True, and I don't really need this because I do layout manually anyway. So I'll remove it in the next PR.